unit-13/javascrip-even/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Javascrip-even</title>
+</head>
+<body>
+    <div id="onmouse">Lesson 1 hover</div><br><br>
+    <input type="text" id="input" placeholder="Lesson 2"><br><br>
+    <input type="number" id="inputnumber" placeholder="lesson 3"><br><br>
```

Comment:
```suggestion
    <input type="number" id="inputNumber" placeholder="lesson 3"><br><br>
```
1 là cách đặt tên theo `camel` đi, hoặc `input-number`